packages/x-tailwindcss/src/x-tailwind-plugin/components/typography/title3/sizes.ts

```diff
@@ -0,0 +1,24 @@
+import { TailwindHelpers } from '../../../../types';
+
+/**
+ * Returns the `sizes` variants for component `title3`.
+ *
+ * @param helpers - The {@link TailwindHelpers} to generate CSS.
+ * @returns The {@link CssStyleOptions} for the variant.
+ */
+// eslint-disable-next-line  @typescript-eslint/explicit-function-return-type
+export function titleSizes({ theme }: TailwindHelpers) {
+  return {
+    sm: {
+      fontSize: theme('fontSize.md')
```

Comment:
Same issue here with the values differences between figma (18px) and the browser (16px)

Reply:
fixed in the designs